java/code/src/com/redhat/rhn/taskomatic/task/RhnJavaJob.java

```diff
@@ -31,37 +33,45 @@
 import java.util.Arrays;
 
 
-
 /**
  * RhnJavaJob
  */
 public abstract class RhnJavaJob implements RhnJob {
 
+    protected static String DEFAULT_PLUGIN_NAME = "File";
```

Comment:
you need a `final` here

Reply:
I can't change the value from within the unit test then. However I doubt the latter is relevant after all :roll_eyes: